src/external/glad.h

```diff
@@ -12,16 +12,16 @@
  *  - ALIAS = False
  *  - DEBUG = False
  *  - HEADER_ONLY = True
- *  - LOADER = True
+ *  - LOADER = False
```

Comment:
I think that statement is not correct (?)

Reply:
Yes it is. raylib doesn't use glad's loader function, but instead `glfwGetProcAddress`. Using glad's loader requires inclusion of `windows.h` which introduces name conflicts for some functions.